### DISCUSSION TITLE
Keep your Filament admin dashboard fast by lazy loading widgets | Rias.be

### DISCUSSION THREAD

**Discussion by giscus** [riasvdv/rias.be#565] [General]
# Keep your Filament admin dashboard fast by lazy loading widgets | Rias.be

Filament is a great admin panel for Laravel built with Livewire, one of its features is creating dashboard widgets to get quick stats of your application.

https://rias.be/blog/keep-your-filament-admin-dashboard-fast-by-lazy-loading-widgets

**Comment by zoparga** [comment]
Hey there,
Thanks for the article, very good!

Just a small correction if someone check this in the future and they scratch they head why it isn't working.
Where you mention the path as `resources/vendor/filament/widgets/stats-overview-widget.blade.php` you forget the `views` subfolder (Laravel 9.x).

The correct path for the widget is:
`resources/views/vendor/filament/widgets/stats-overview-widget.blade.php` 

Have a nice day!
